packages/maskbook/src/settings/settings.ts

```diff
@@ -77,6 +77,10 @@ export const lastActivatedWalletProvider = createInternalSettings<ProviderType>(
     ProviderType.Maskbook,
 )
 
+export const isMetaMaskUnlocked = createGlobalSettings<boolean>('is metaMask unlocked', false, {
```

Comment:
Do this setting needed to be shared across multiple contexts?

Reply:
Currently, it is set by background-script, and used only by front-end plugin.